LayoutTests/webanimations/document-teardown-due-to-last-reference-crash-expected.txt

```diff
@@ -0,0 +1,3 @@
+FAIL: Timed out waiting for notifyDone to be called
```

Comment:
This doesn't seem like a good output.

Reply:
Indeed, this is the test we've managed to reduce to for a much more complex test and it times out in the non-crashing case. I have not managed to get it to a non-timeout state.

Reply:
I tried changing the end of this test to:

```
    animationWeakRef.deref().ready.then(() => globalThis.testRunner?.notifyDone());
    GCController.collect();
```

But the test still times out.

Reply:
I believe we do have a working document to call `notifyDone()` on since the one that is being torn down is the one created with `new Document`.

Reply:
You're right. I don't fully understand how this test works to be honest, but tried as I might I couldn't find a way to get it so that it would yield the crash without the patch but also allow for `testRunner.notifyDone()` to be called with the patch. I'm going to land as-is for now, and try harder to make a test that can complete without a timeout.